source/platform/renderers/Allegro5/Allegro5.cpp

```diff
@@ -126,7 +129,7 @@ Font::Status Allegro::LoadFont(const Font& font)
     {
         ALFontData fontData;
         fontData.font = deleted_unique_ptr<ALLEGRO_FONT>(afont, [](ALLEGRO_FONT* mem) { if (mem) al_destroy_font(mem); });
-        m_fonts.insert(std::make_pair(font, std::move(fontData)));
+        m_lastFont = &(*m_fonts.insert(std::make_pair(font, std::move(fontData))).first);
```

Comment:
`std::move` has no effect here since you have already constructed `fontData`?

Reply:
`fontData` cannot be copied, the `std::move` forces the move constructor to be called, swapping the properties between the constructed `fontData` and the unconstructed one in `std::pair`.